pkg/tuple/comparison_test.go

```diff
@@ -71,6 +71,8 @@ func TestEqual(t *testing.T) {
 		MustParse("document:foo#viewer@user:tom[somecaveat:{\"hi\":{\"yo\":123}}]"),
 		MustParse("document:foo#viewer@user:tom[somecaveat:{\"hi\":{\"yo\":{\"hey\":true}}, \"hi2\":{\"yo2\":{\"hey2\":false}}}]"),
 		MustParse("document:foo#viewer@user:tom[somecaveat:{\"hi\":{\"yo\":{\"hey\":true}}, \"hi2\":{\"yo2\":{\"hey2\":[1,2,3]}}}]"),
+		MustParse("document:foo#viewer@user:tom[expiration:2020-01-01T00:00:00Z]"),
+		MustParse("document:foo#viewer@user:tom[somecaveat:{\"hi\":\"there\"}][expiration:2020-01-01T00:00:00Z]"),
```

Comment:
Are we doing separate brackets for each trait? I'd be in favor of having them within the same set of brackets and comma-separated like in the first example from slack.

Reply:
Yeah, I felt each bracket made it more readable